### PR TITLE
Adds documentation for log type categories

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
Categories have been added to log types for easier filtering and sorting.  

### Issues Resolved
Fixes https://github.com/opensearch-project/documentation-website/issues/5147

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
To help with the tech/doc review, here are some of the changes: 

On Supported log types topic:
- In the first paragraph I added the info about the new category feature
- The second paragraph was taken from the 'Create custom log detector' page. I just added the bit about the Category column
- New column for 'Category' in the table 
- New Security > Waf log type description
- Page actions was taken from 'Create detector' topic, so the only new text is the last bullet about the category.

In the 'Create custom..' topic:
- Much of the content was moved, as noted above, to the other page where it was more relevant.
- Simplified the steps and added the part about selecting a category.
- I also added links back to the log types page.

**Review by praveensameneni — APPROVED** [review]

File: _security-analytics/sec-analytics-config/log-types.md [hunk 1]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+
+To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, it's category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected, and where you can filter the list by the category. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/c-log-type.png" alt="The Log types landing page." width="85%">
+
+The following table shows the log types that are currently supported by Security Analytics for ingestion, mapping, and monitoring. 
+
+| Category | Log type | Description | 
+| :--- |:--- |:--- |
+| Access Management | `Ad_ldap` | Active Directory logs that track such things as LDAP queries, errors from the LDAP server, time-out events, and unsecure LDAP binds. |
```
Comment (jimishs) [inline _security-analytics/sec-analytics-config/log-types.md:21]: previously it was called AD / LDAP logs. Why did we rename it to "Ad_ldap"?

File: _security-analytics/sec-analytics-config/log-types.md [hunk 2]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Access Management | `Apache_access` | A log type that is responsible for recording data for all requests processed by an Apache HTTP server. |
```
Comment (jimishs) [inline _security-analytics/sec-analytics-config/log-types.md:22]: Is there a reason that we are renaming the log types to be shorter and removing proper case for each? Why not call it Apache Access ?

File: _security-analytics/sec-analytics-config/log-types.md [hunk 3]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Access Management | `Okta` | These logs record Okta events from a range of actions such as downloading an export file, requesting an application access, or revoking privileges. |
+| Applications | `GitHub` | Logs that monitor workflows created by [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). |
+| Applications| `Gworkspace` | Logs for Google Workspace that can monitor log entries such as admin actions, group and group membership actions, and events having to do with logging in. |  
+| Applications| `M365` | Microsoft 365 audit logs collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity. |
+| Cloud Services | `Azure` | Logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services. |
```
Comment (jimishs) [inline _security-analytics/sec-analytics-config/log-types.md:27]: Azure isnt descriptive. Can we rename this appropriately? Whats the exact log type we are monitoring?

File: _security-analytics/sec-analytics-config/log-types.md [hunk 4]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Cloud Services | `CloudTrail` | Logs that monitor events for an AWS CloudTrail account. OpenSearch can ingest CloudTrail log data from both [AWS Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (S3) accounts and [Amazon Security Lake](https://docs.aws.amazon.com/security-lake/latest/userguide/what-is-security-lake.html) service accounts. |
```
Comment (jimishs) [inline _security-analytics/sec-analytics-config/log-types.md:28]: AWS CloudTrail is the proper name

File: _security-analytics/sec-analytics-config/log-types.md [hunk 5]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Applications | `GitHub` | Logs that monitor workflows created by [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). |
+| Applications| `Gworkspace` | Logs for Google Workspace that can monitor log entries such as admin actions, group and group membership actions, and events having to do with logging in. |  
+| Applications| `M365` | Microsoft 365 audit logs collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity. |
+| Cloud Services | `Azure` | Logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services. |
+| Cloud Services | `CloudTrail` | Logs that monitor events for an AWS CloudTrail account. OpenSearch can ingest CloudTrail log data from both [AWS Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (S3) accounts and [Amazon Security Lake](https://docs.aws.amazon.com/security-lake/latest/userguide/what-is-security-lake.html) service accounts. | 
+| Cloud Services | `S3` | These logs track requests for access to an Amazon S3 bucket. |
```
Comment (jimishs) [inline _security-analytics/sec-analytics-config/log-types.md:29]: Amazon S3. For all the descriptions, can we be more specific on what specific logs are we capturing? Is this referring to Amazon S3 audit logs?

File: _security-analytics/sec-analytics-config/log-types.md [hunk 6]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Network Activity| `Dns` | A log that stores Domain Name System (DNS) activity. Security Analytics supports [Open Cybersecurity Schema Framework](https://docs.aws.amazon.com/security-lake/latest/userguide/open-cybersecurity-schema-framework.html) (OCSF) log and event data, which includes the Security Lake log type Route 53. Given that Route 53 is a log that captures DNS activity, its log type should be specified as DNS logs when defining a detector. |
```
Comment (jimishs) [inline _security-analytics/sec-analytics-config/log-types.md:30]: DNS. Not sure why we talk about OCSF here. The assumption is that this table doesnt care about the specific format (eg: OCSF, ECS) . It is a table that mainly talks about the supported log types and categories

File: _security-analytics/sec-analytics-config/log-types.md [hunk 7]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Network Activity | `Network` | A log that records events that happen in a system's network, such as login attempts and application events. | 
+| Network Activity | `vpcflow` | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) capture information about the IP traffic going to and from network interfaces in your VPC. |
+| Security | `Waf` |  The role of WAF (Web Application Firewall) is to monitor and filter HTTP traffic between a web application and the internet. It is tasked wth preventing common security attacks such as cross-site scripting (XSS), SQL Injection (SQi), and so on. This new log type (introduced in OpenSearch 2.11) is for users that require monitoring for the WAF use case out-of-the box from the Security plugin. |
```
Comment (jimishs) [inline _security-analytics/sec-analytics-config/log-types.md:33]: we need to rename almost all the log types and use proper case and acronyms, including updating description with the detailed log types we support

File: _security-analytics/sec-analytics-config/log-types.md [hunk 8]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| System Activity | `Linux` | Logs that record events on Linux. |
```
Comment (jimishs) [inline _security-analytics/sec-analytics-config/log-types.md:34]: We need to be more specific. What linux logs are we talking about?

**Review by jimishs — CHANGES_REQUESTED** [review]

File: _security-analytics/sec-analytics-config/log-types.md (hunk 8 above)
Comment (praveensameneni) [reply _security-analytics/sec-analytics-config/log-types.md:34]: syslog

**Comment by hdhalter** [comment]
I've added a new column called "Log names" with the full names. Can someone please provide more detailed descriptions of the log types? Thanks.

**Review by sbcd90 — APPROVED** [review]

**Comment by hdhalter** [comment]
@jimishs - I removed the Log Name column. Can you please approve? Thanks.

File: _security-analytics/sec-analytics-config/log-types.md [hunk 9]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+
+To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, it's category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected, and where you can filter the list by the category. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/c-log-type.png" alt="The Log types landing page." width="85%">
+
+The following table shows the log types that are currently supported by Security Analytics for ingestion, mapping, and monitoring. 
+
+| Category | Log type | Description | 
+| :--- |:--- |:--- 
+| Access Management | `Ad_ldap` | Active Directory logs that track such things as LDAP queries, errors from the LDAP server, time-out events, and unsecure LDAP binds. |
```
Comment (kolchfa-aws) [inline _security-analytics/sec-analytics-config/log-types.md:21]:
```suggestion
| Access Management | `Ad_ldap` | Active Directory logs that track LDAP queries, errors from the LDAP server, timeout events, and unsecure LDAP binds. |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 10]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+Logs contain raw data about events that happen throughout a system and in its separate parts. As of OpenSearch 2.11, log types are grouped by category to help select, filter, and search the log types. 
+
+To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, it's category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected, and where you can filter the list by the category. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/c-log-type.png" alt="The Log types landing page." width="85%">
+
+The following table shows the log types that are currently supported by Security Analytics for ingestion, mapping, and monitoring. 
+
+| Category | Log type | Description | 
+| :--- |:--- |:---
```
Comment (kolchfa-aws) [inline _security-analytics/sec-analytics-config/log-types.md:20]:
```suggestion
| :--- |:--- |:--- |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 11]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, it's category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected, and where you can filter the list by the category. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/c-log-type.png" alt="The Log types landing page." width="85%">
+
+The following table shows the log types that are currently supported by Security Analytics for ingestion, mapping, and monitoring. 
+
+| Category | Log type | Description | 
+| :--- |:--- |:--- 
+| Access Management | `Ad_ldap` | Active Directory logs that track such things as LDAP queries, errors from the LDAP server, time-out events, and unsecure LDAP binds. |
+| Access Management | `Apache_access` | Apache Access logs that record data for all requests processed by an Apache HTTP server. |
```
Comment (kolchfa-aws) [inline _security-analytics/sec-analytics-config/log-types.md:22]:
```suggestion
| Access Management | `Apache_access` | Apache access logs that record data for all requests processed by an Apache HTTP server. |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 12]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Cloud Services | `S3` | AWS S3 logs that track requests for access to an Amazon S3 bucket. |
+| Network Activity| `Dns` | DNS logs that store Domain Name System (DNS) activity. |
+| Network Activity | `Network` | Network logs that records events that happen in a system's network, such as login attempts and application events. | 
+| Network Activity | `vpcflow` | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) that capture information about the IP traffic going to and from network interfaces in your VPC. |
+| Security | `Waf` | Web Application Firewall (WAF) logs (introduced in OpenSearch 2.11) for users that require monitoring of the WAF use case that's provided out-of-the box from the Security plugin. The role of WAF is to monitor and filter HTTP traffic between a web application and the internet. It is tasked wth preventing common security attacks such as cross-site scripting (XSS), SQL Injection (SQi), and so on. |
```
Comment (kolchfa-aws) [inline _security-analytics/sec-analytics-config/log-types.md:33]:
```suggestion
| Security | `Waf` | Web Application Firewall (WAF) logs (introduced in OpenSearch 2.11) for users that require monitoring of the WAF use case that's provided out-of-the box in the Security plugin. The role of WAF is to monitor and filter HTTP traffic between a web application and the internet. WAF prevents common security attacks, such as cross-site scripting (XSS), SQL Injection (SQi), and so on. |
```

**Review by kolchfa-aws — APPROVED** [review]
LGTM with comments

File: _security-analytics/sec-analytics-config/custom-log-type.md [hunk 13]
```diff
@@ -8,34 +8,20 @@ nav_order: 18
...
-
+Log types represent the different sources of data used for threat detection in Security Analytics. In addition to the standard [log types]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/log-types/) supported by Security Analytics, you can create custom log types for your threat detectors.
 
 ## Creating a custom log type
 
-After selecting **Create log type** on the **Log types** page, the **Create log type** page opens and provides the necessary fields for creating a new log type:
-
-1. Enter a name for the log type.
+To create a custom log type: 
+1. From the dashboard, select **OpenSearch Plugins** > **Security Analytics**, then select **Detectors** > **Log types**.
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/custom-log-type.md:16]:
```suggestion
1. From the dashboard, select **OpenSearch Plugins** > **Security Analytics**, and then select **Detectors** > **Log types**.
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 14]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
-| **Microsoft Azure logs** | Logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services.
-| **VPC Flow logs** | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) capture information about the IP traffic going to and from network interfaces in your VPC.
+Logs contain raw data about events that happen throughout a system and in its separate parts. As of OpenSearch 2.11, log types are grouped by category to help select, filter, and search the log types. 
+
+To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, its category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected, and the **Category** filter you can use to filter the list by the category.
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:13]:
```suggestion
To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, its category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected and the **Category** filter you can use to filter the list by the category. 
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 15]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/c-log-type.png" alt="The Log types landing page." width="85%">
+
+The following table shows the log types that are currently supported by Security Analytics for ingestion, mapping, and monitoring. 
+
+| Category | Log type | Description | 
+| :--- |:--- |:--- |
+| Access Management | `Ad_ldap` | Active Directory logs that track LDAP queries, errors from the LDAP server, timeout events, and unsecure LDAP binds. |
+| Access Management | `Apache_access` | Apache access logs that record data for all requests processed by an Apache HTTP server. |
+| Access Management | `Okta` | Okta logs that record Okta events from a range of actions such as downloading an export file, requesting an application access, or revoking privileges. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:23]:
```suggestion
| Access Management | `Okta` | Okta logs that record Okta events from a range of actions, such as downloading an export file, requesting application access, or revoking privileges. |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 16]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Applications | `GitHub` | GitHub logs that monitor workflows created by [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). |
+| Applications| `Gworkspace` | Google Workspace logs that monitor log entries such as admin actions, group and group membership actions, and events having to do with logging in. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:25]:
```suggestion
| Applications| `Gworkspace` | Google Workspace logs that monitor log entries such as admin actions, group and group membership actions, and events related to logging in. |  
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 17]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+
+| Category | Log type | Description | 
+| :--- |:--- |:--- |
+| Access Management | `Ad_ldap` | Active Directory logs that track LDAP queries, errors from the LDAP server, timeout events, and unsecure LDAP binds. |
+| Access Management | `Apache_access` | Apache access logs that record data for all requests processed by an Apache HTTP server. |
+| Access Management | `Okta` | Okta logs that record Okta events from a range of actions such as downloading an export file, requesting an application access, or revoking privileges. |
+| Applications | `GitHub` | GitHub logs that monitor workflows created by [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). |
+| Applications| `Gworkspace` | Google Workspace logs that monitor log entries such as admin actions, group and group membership actions, and events having to do with logging in. |  
+| Applications| `M365` | Microsoft 365 audit logs that collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:26]:
```suggestion
| Applications| `M365` | Microsoft 365 audit logs that collect a range of data for Microsoft 365, including records from call details, performance data, SQL Server, security events, and access control activity. |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 18]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Cloud Services | `Azure` | Microsoft Azure logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:27]:
```suggestion
| Cloud Services | `Azure` | Microsoft Azure logs that monitor log data for cloud applications managed by Azure Cloud Services. |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 19]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Cloud Services | `CloudTrail` | AWS CloudTrail logs that monitor events for an AWS CloudTrail account. OpenSearch can ingest CloudTrail log data from both [AWS Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (S3) accounts and [Amazon Security Lake](https://docs.aws.amazon.com/security-lake/latest/userguide/what-is-security-lake.html) service accounts. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:28]:
```suggestion
| Cloud Services | `CloudTrail` | AWS CloudTrail logs that monitor events for an AWS CloudTrail account. OpenSearch can ingest CloudTrail log data from both [Amazon Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (Amazon S3) accounts and [Amazon Security Lake](https://docs.aws.amazon.com/security-lake/latest/userguide/what-is-security-lake.html) service accounts. | 
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 20]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Applications| `Gworkspace` | Google Workspace logs that monitor log entries such as admin actions, group and group membership actions, and events having to do with logging in. |  
+| Applications| `M365` | Microsoft 365 audit logs that collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity. |
+| Cloud Services | `Azure` | Microsoft Azure logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services. |
+| Cloud Services | `CloudTrail` | AWS CloudTrail logs that monitor events for an AWS CloudTrail account. OpenSearch can ingest CloudTrail log data from both [AWS Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (S3) accounts and [Amazon Security Lake](https://docs.aws.amazon.com/security-lake/latest/userguide/what-is-security-lake.html) service accounts. | 
+| Cloud Services | `S3` | AWS S3 logs that track requests for access to an Amazon S3 bucket. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:29]:
```suggestion
| Cloud Services | `S3` | Amazon S3 logs that track requests for access to an S3 bucket. |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 21]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Network Activity| `Dns` | DNS logs that store Domain Name System (DNS) activity. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:30]:
```suggestion
| Network Activity| `Dns` | DNS logs that store DNS activity. |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 22]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Network Activity | `Network` | Network logs that record events that happen in a system's network, such as login attempts and application events. | 
+| Network Activity | `vpcflow` | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) that capture information about the IP traffic going to and from network interfaces in your VPC. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:32]:
```suggestion
| Network Activity | `vpcflow` | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) that capture information about the IP traffic going to and from network interfaces in your virtual private cloud (VPC). |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 23]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Security | `Waf` | Web Application Firewall (WAF) logs (introduced in OpenSearch 2.11) for users that require monitoring of the WAF use case that's provided out-of-the box in the Security plugin. The role of WAF is to monitor and filter HTTP traffic between a web application and the internet. WAF prevents common security attacks, such as cross-site scripting (XSS), SQL Injection (SQi), and so on. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:33]:
```suggestion
| Security | `Waf` | Web Application Firewall (WAF) logs (introduced in OpenSearch 2.11) for users that require monitoring of the WAF use case that's provided out of the box in the Security plugin. The role of WAF is to monitor and filter HTTP traffic between a web application and the internet. WAF prevents common security attacks, such as cross-site scripting (XSS) and SQL Injection (SQi). |
```

Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:33]: Are we referring to AWS WAF here? If so, the service name must always be "AWS WAF", never shortened to "WAF", and we don't define WAF as "Web Application Firewall". Instead, we say "AWS WAF is a web application firewall."

File: _security-analytics/sec-analytics-config/log-types.md [hunk 24]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| System Activity | `Linux` | Linux System logs that record syslog events on Linux. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:34]:
```suggestion
| System Activity | `Linux` | Linux system logs that record Linux syslog events. |
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 25]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+
+To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, its category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected, and the **Category** filter you can use to filter the list by the category. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/c-log-type.png" alt="The Log types landing page." width="85%">
+
+The following table shows the log types that are currently supported by Security Analytics for ingestion, mapping, and monitoring. 
+
+| Category | Log type | Description | 
+| :--- |:--- |:--- |
+| Access Management | `Ad_ldap` | Active Directory logs that track LDAP queries, errors from the LDAP server, timeout events, and unsecure LDAP binds. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:21]: Confirm sentence case for the first column.

File: _security-analytics/sec-analytics-config/log-types.md [hunk 26]
```diff
@@ -8,22 +8,44 @@ nav_order: 16
...
+| Network Activity| `Dns` | DNS logs that store Domain Name System (DNS) activity. |
+| Network Activity | `Network` | Network logs that record events that happen in a system's network, such as login attempts and application events. | 
+| Network Activity | `vpcflow` | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) that capture information about the IP traffic going to and from network interfaces in your VPC. |
+| Security | `Waf` | Web Application Firewall (WAF) logs (introduced in OpenSearch 2.11) for users that require monitoring of the WAF use case that's provided out-of-the box in the Security plugin. The role of WAF is to monitor and filter HTTP traffic between a web application and the internet. WAF prevents common security attacks, such as cross-site scripting (XSS), SQL Injection (SQi), and so on. |
+| System Activity | `Linux` | Linux System logs that record syslog events on Linux. |
+| System Activity | `Windows` | Windows logs that record events that have happened in the operating system, applications, and other system services for Windows. |
```
Comment (natebower) [inline _security-analytics/sec-analytics-config/log-types.md:35]:
```suggestion
| System Activity | `Windows` | Windows logs that record events that have happened in the operating system, applications, and other Windows system services. |
```

**Review by natebower — COMMENTED** [review]
@hdhalter Please see my comments and changes and let me know if you have any questions. Thanks!

**Comment by hdhalter** [comment]
@natebower - I've accepted your revisions. Thanks! I am just waiting on confirmation from development about AWS WAF.

**Pull request merged**